extra/man/alacritty.5.scd

```diff
@@ -24,6 +26,13 @@ This section documents the root level of the configuration file.
 	All imports must either be absolute paths starting with _/_, or paths
 	relative to the user's home directory starting with _~/_.
 
+	Example:
+
+		import = [++
+  _"~/.my_fancy_colorscheme.toml"_,++
```

Comment:
This tilde will be transformed into one that cannot be used in the shell.

Reply:
What do you mean, we support `~` and I've tested this import explicitly.

Reply:
When generating this with scdoc, I get the following:

```
"˜/.my_fancy_colorscheme.toml"
```

This does **not** work.

Reply:
As discussed, it's an issue with `groff`.